backend_addon/{{ cookiecutter.__folder_name }}/.gitignore

```
@@ -7,14 +7,16 @@
 *.pyo
 
 # tools related
-build/
+__pycache__/
+.coverage
 .coverage
+.mypy_cache
+.pytest_cache
+.ruff_cache
+build/
```

Excluding generic names like this can be annoying, so make sure it's only at the root of the repo:

```suggestion
/build/
```